src/extension.ts

```diff
@@ -63,3 +86,76 @@ const getComponentName = (): Thenable<string | undefined> =>
         : "Please enter a valid component name";
     },
   });
+
+/**
+ * Check availability of Angular CLI on the command line
+ * @param directory Directory which will use the Angular CLI
+ */
```

Comment:
You should rename the function to `isAngularCliAvailable`, as thats what you describe in the comment.

Reply:
Renamed it, wasn't sure how to name it properly but this version sounds definitely better.